### PR TITLE
Allow to use replaceWhere without partitioning

### PR DESCRIPTION
**Pull request by hanna-liashchuk**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -33,7 +33,7 @@ headerLicense := Some(
   )
 )
 name := "DeltaFlow"
-version := "1.0.1"
+version := "1.0.2"
```
Comment (VShkaberda) [inline build.sbt:36]: Shouldn't version bumping be separated from PR delivering changes? The way it is in [requests](https://github.com/psf/requests/pull/6182/commits/82e63a7c60b707d4d31a8a88416738fd66098c97) via separate PR or in [wvlet/airframe](https://github.com/wvlet/airframe/commit/25774b1fb69727c962d22fce93be514b005118fe) via tags. I don't know which way is better though.

File: src/main/scala/palefat/data/mirror/Runner.scala [hunk 2]
```diff
@@ -77,7 +77,7 @@ object Runner extends LogSupport {
         FilterBuilder.buildReplaceWherePredicate(
           ds,
           config.lastPartitionCol,
```
Comment (VShkaberda) [inline src/main/scala/palefat/data/mirror/Runner.scala]: Is it right that Z-ordering is executed using `partitionCols.last`?

Comment (hanna-liashchuk) [reply src/main/scala/palefat/data/mirror/Runner.scala]: this is used to filter data. and yes, if it's not a partition column, it's failing with error #15 
z-order itself is executed on a column specified in zorderby_col, a partition is used only to run in on a subset of data

File: build.sbt (hunk 1 above)
Comment (hanna-liashchuk) [reply build.sbt:36]: I guess we could use sbt-release?

**Review by alexsukhrin — APPROVED** [review]

**Review by VShkaberda — APPROVED** [review]

**Pull request merged**